python/lsst/afw/cameraGeom/_amplifier.py

```diff
@@ -181,6 +181,20 @@ def transform(self, *, outOffset=None, outFlipX=False, outFlipY=False):
         bbox.shift(self.getRawXYOffset() + shift - outOffset)
 
         getattr(self, f"setRaw{bboxName}BBox")(bbox)
+
+    # Update the Readout Corner if we've flipped anything.
+    outReadoutCorner = self.getReadoutCorner()
+    if self.getRawFlipX() != outFlipX:
+        xFlipMapping = {ReadoutCorner.LL: ReadoutCorner.LR, ReadoutCorner.LR: ReadoutCorner.LL,
```

Comment:
Would it make more sense to do this operation in L166, so it happens right after `bbox` is flipped?